deploy/kessel-relations-deploy.yaml

```diff
@@ -32,13 +32,12 @@ objects:
     kind: ClowdApp
     metadata:
       name: ${CLOWDAPP_NAME}
-      namesapce: ${NAMESPACE}
     spec:
       envName: ${ENV_NAME}
       testing:
         iqePlugin: relations_api
       deployments:
-        - name: relations
+        - name: api
```

Comment:
api seems to generic - should we rename it to relations-api. given inventory and other components are there as a part of kessel? WDYT?

Reply:
The pods will appear as `ClowdApp.name + "-" + Deployment.name` in OpenShift, so `kessel-relations-api` in this case. `api` is only the name of one of the deployments from the `relations-kessel` ClowdApp.

Reply:
FYI the exact same naming convention is already used for `inventory-api` in the kessel-stage namespace, if you want to see how it looks.

Reply:
perfect thanks for the clarification